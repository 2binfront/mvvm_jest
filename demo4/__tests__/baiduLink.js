import BaiduLink from "../baiduLink";

import renderer from 'react-test-renderer';
import React from "react";

test('baidu render correctly',() => {
    const tree = renderer.create(<BaiduLink></BaiduLink>);
    expect(tree).toMatchSnapshot();
})