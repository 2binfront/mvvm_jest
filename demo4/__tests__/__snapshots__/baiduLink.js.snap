// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`baidu render correctly 1`] = `
<a
  href="https://www.baidu.com"
>
  百度一下
</a>
`;
